tools/coqdoc/cpretty.mll

```diff
@@ -742,8 +748,10 @@ and doc_bol = parse
       }
   | (space_nl* nl) as s
       { new_lines (count_newlines s) lexbuf; Output.paragraph (); doc_bol lexbuf }
-  | "<<" space*
-      { Output.start_verbatim false; verbatim 0 false lexbuf; doc_bol lexbuf }
+  | "<<" space* nl
+      { new_lines 1 lexbuf; Output.start_verbatim false; verbatim_block lexbuf; doc_bol lexbuf }
+  | "<<"
+      { Output.start_verbatim true; verbatim_inline lexbuf; doc_bol lexbuf }
```

Comment:
After `verbatim_inline`, we probably want `doc` instead of `doc_bol`.

```suggestion
      { Output.start_verbatim true; verbatim_inline lexbuf; doc None lexbuf }
```

For example, in `(** <<code>> - bar *)`  the part after `<<code>>` should not be parsed as a list, but as the continuation of the line.